hack/build/setup-pac-integration.sh

```diff
@@ -82,8 +82,14 @@ if [ -n "${PAC_GITLAB_TOKEN}" ]; then
 fi
 
 oc create namespace -o yaml --dry-run=client ${PAC_NAMESPACE} | oc apply -f-
+# short term, we create the old pipelines-as-code namespace until we migrate through prod to the new location; then we will go back
+# and switch all the shared/based overlay refs to point to openshift-pipelines, and can remove this
+oc create namespace -o yaml --dry-run=client pipelines-as-code | oc apply -f-
```

Comment:
This is not problem, staging is not using setup-pac-integration.sh for setup, it's using external secrets, so you need to update deployment of external secret for stage to be into right namespace -> https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/base/external-secrets/pipelines-as-code/kustomization.yaml#L5

Reply:
update pushed, along with the configmap patch needing to be updated I believe @Michkov PTAL